azure-cli2017.pyproj

```diff
@@ -104,6 +104,10 @@
     <Compile Include="command_modules\azure-cli-acr\azure\cli\command_modules\acr\tests\latest\test_acr_commands.py" />
     <Compile Include="command_modules\azure-cli-acr\azure\cli\command_modules\acr\tests\latest\test_acr_commands_mock.py" />
     <Compile Include="command_modules\azure-cli-acr\azure\cli\command_modules\acr\webhook.py" />
+	<Compile Include="command_modules\azure-cli-acr\azure\cli\command_modules\acr\build.py" />
```

Comment:
What does this pyproj do? Do we need to include all our other files?
NIT: indenting

Reply:
This proj is required only when we want to open it in vs2017

Reply:
@jaysterp This might be the issue you were facing the other day when trying to run on VS17?